Plugins/Hashing/include/Acts/Plugins/Hashing/HashingAnnoy.ipp

```diff
@@ -0,0 +1,128 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Definitions/Units.hpp"
+
+#include <map>
+#include <set>
+#include <vector>
+
+#include <annoy/annoylib.h>
```

Comment:
Due to template functions, I did not manage to convert the ipp files into cpp files and hide the headers there.
Is it good enough for you @andiwand @paulgessinger ?

Reply:
In case there is no way around this I would just accept it

Reply:
Nothing I tried worked. Maybe this could be in another PR as I have no idea on how to make it work right now.